### ISSUE TITLE
View logs in Elyra

### ISSUE THREAD

**Issue by Shreyanand** [operate-first/support#145]
**Is your feature request related to a problem? Please describe.**
In order to debug pipelines while automating my workflow, I'd need to see Jupyter lab logs. Currently, when I go to log console I don't the see the logs there. When I submit a faulty pipeline, I can see a traceback of the error added in Elyra v06 which is very helpful. However, for a detailed analysis, should the logs be available in the log console as well? @pacospace

**Comment by pacospace** [comment]
> **Is your feature request related to a problem? Please describe.**
> In order to debug pipelines while automating my workflow, I'd need to see Jupyter lab logs. Currently, when I go to log console I don't the see the logs there. When I submit a faulty pipeline, I can see a traceback of the error added in Elyra v06 which is very helpful. However, for a detailed analysis, should the logs be available in the log console as well? @pacospace

I had a look upstream and as of now for local run of pipeline there is an open feature request: https://github.com/elyra-ai/elyra/issues/1265

We have to wait for that feature to be available, but in Elyra v0.0.6 it is more helpful indeed:
![Screenshot from 2021-03-23 17-24-57](https://user-images.githubusercontent.com/27498679/112182482-9d7af580-8bfd-11eb-8fd1-bc8070dbc1ba.png)

**Comment by Shreyanand** [comment]
![Screenshot from 2021-03-23 15-47-28](https://user-images.githubusercontent.com/8916126/112208967-f7e87600-8c3e-11eb-88fd-3fcf84832811.png)

@pacospace Got an error with no traceback in Elyra v0.0.6. Is it the kind of error that determines if we get a traceback or not?

**Comment by pacospace** [comment]
> ![Screenshot from 2021-03-23 15-47-28](https://user-images.githubusercontent.com/8916126/112208967-f7e87600-8c3e-11eb-88fd-3fcf84832811.png)
> 
> @pacospace Got an error with no traceback in Elyra v0.0.6. Is it the kind of error that determines if we get a traceback or not?

Maybe having public view on pod/logs might be good, I saw discussion on Slack cc @tumido

**Comment by tumido** [comment]
Thread link: https://operatefirst.slack.com/archives/C01RMPVUUK1/p1616437865005500

We will create a rolebinding for that :+1:

**Comment by tumido** [comment]
@pacospace does it make sense to move discussions on this to upstream? I'm not sure how much more besides https://github.com/operate-first/apps/issues/428 we can help with this in here

**Comment by pacospace** [comment]
> @pacospace does it make sense to move discussions on this to upstream? I'm not sure how much more besides [operate-first/apps#428](https://github.com/operate-first/apps/issues/428) we can help with this in here

Yes definitely, maybe the issue I linked above is slightly different but also a feature that is interesting to have in Elyra I guess.

@Shreyanand can you please open an issue upstream in https://github.com/elyra-ai/elyra/issues/new?assignees=&labels=component%3Apipeline-editor%2C+component%3Apipeline-runtime%2C+status%3ANeeds+Triage%2C+kind%3Auser-feedback&template=pipeline-issue-template.md&title=
let me know if you need support on that

**Comment by Shreyanand** [comment]
issue on upstream: https://github.com/elyra-ai/elyra/issues/1498

**Comment by ptitzler** [comment]
We/Elyra have improved error handling, which should expose the relevant information in the dialog. If you encounter error messages like the ones shown above using the _0.0.7 image_  let us know and we'll improve them. In many cases this cryptic JSON error is masking a runtime configuration issue, which is typically caused by invalid host/port/namespace/user values.
